src/explore-education-statistics-common/src/modules/table-tool/components/__tests__/__data__/TableHeadersConfig.data.ts

```diff
@@ -0,0 +1,95 @@
+import {
```

Comment:
Test data file names are typically camelCased as they don't refer to a single thing e.g. `tableHeadersConfig.data.ts`